packages/kotti-ui/source/kotti-style/index.scss

```diff
@@ -6,7 +6,7 @@
 @import 'transitions';
 
 // Reset and dependencies
-@import '~normalize.css';
+// @import '~normalize.css';
```

Comment:
@FlorianWendelborn don't we need normalize here?

Reply:
@mvangent `normalize.css` is now imported in `index.ts`